### DISCUSSION TITLE
Using Airbyte Destination Data in LangChain

### DISCUSSION THREAD

**Discussion by mendonk** [airbytehq/airbyte#35245] [General]
This might be a pretty dumb question but I can't figure it out, sorry.
I have a little test running (in cloud.airbyte.com)- 
* GNews API as a source
* AstraDB vector database as destination
* Connector default between the two
The test works, and I find Airbyte pretty nice to use.
My problem is once I want to use LangChain (for example) with the data in the destination - there's no `page_content` or `metadata` in the destination, just the schema from Gnews, so Langchain/AstraDB don't know what to do. How can I add the Airbyte data to Documents for langchain to use?
Thank you.

**Comment by marcosmarxm** [comment]
Hello @mendonk did you check this article? https://airbyte.com/tutorials/chat-with-your-data-using-openai-pinecone-airbyte-and-langchain maybe can give you some hints. Not sure if all connectors are going to work out-of-box to send data to Langchain (I need do some testing though :D)

[a comment by an automated account is not shown]
